data/korath/korath outfits.txt

```diff
@@ -91,6 +92,33 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Liquid Sodium Cooler"
+	category "Systems"
+	cost 1660000
+	thumbnail "outfit/liquid sodium"
+	"mass" 34
+	"outfit space" -34
+	"heat capacity" 680
```

Comment:
I do like the idea of the Korath using heat capacity. It's as if they've given up on getting rid of the heat entirely and instead are just looking for more places to put it. Admittedly given that this is such a new attribute though, I'm not sure on how effective it'll be; these values could potentially be too high, but I'll let them go for now and see how things play out.

Reply:
I agree, it adds some nice differentiation from the usual "just try to get cold" all the time.

Reply:
>I'm not sure on how effective it'll be; these values could potentially be too high, but I'll let them go for now and see how things play out.

I played around with it for a good while and it indeed is a bit weird compared to standard approaches to cooling, but in general, even with switching up it's weighting it's generally a trade-off of much higher base running heat % but eliminating the risk of overheating on your own, at the cost of a decent chunk of even Korath energy generation.

It ends up being more a supplement than a larger replacement cooling. Running one instead of an equal weight in heat shunts is generally a win, as long as you're still running at least a few heat shunts alongside it, and I liked how that felt. :)

Reply:
From what I gathered in the Discord discussion last night that rather went over my head, it sounds like this is also offering slightly better protection against heat attacks than active cooling, at the expense of constant energy use rather than scaling

Reply:
@Amazinite in terms of effectiveness, `6 * "heat capacity" * "heat dissipation"` gives the amount by which it will increase 'max heat' (as displayed in game), so it would be equivalent to that much cooling (except it doesn't actually reduce the heat in the ship so it is no less vulnerable to infrared tracking).
On a Raider, 680 heat capacity is equivalent to 2040 cooling (heat dis = 0.5). On something like a Mereti ship, or a very small ship, it could be over 4000.

Using 0.5 heat dissipation:

name | creator | size | cooling | active cooling | cooling energy | energy consumption | heat cap | cooling/ton | active/ton | total/ton | active/energy
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Small Heat Shunt | Korath | 7 | 13.5 | 0 | 0 | 0 | 0 | 1.92857142857143 | 0 | 1.92857142857143 | 0
Large Heat Shunt | Korath | 24 | 54 | 0 | 0 | 0 | 0 | 2.25 | 0 | 2.25 | 0
Thermoelectric Cooler | Remnant | 4 | 0 | 8 | 0.4 | 0 | 0 | 0 | 2 | 2 | 20
Cooling Module | Coalition | 5 | 0 | 21 | 1.4 | 0 | 0 | 0 | 4.2 | 4.2 | 15
Large Sodium Cooler | Korath Exiles | 34 | 0 | 116 | 5.2 | 2.4 | 680 | 0 | 3.41176470588235 | 3.41176470588235 | 15.2631578947368
Large Sodium Cooler effective (0.5 heat dissipation) | Korath Exiles | 34 | 0 | 150 | 7.6 | 0 | 0 | 0 | 4.41176470588235 | 4.41176470588235 | 19.7368421052632


Considering the active cooling component alone (116 for 5.2 energy), we get an efficiency of just over 22 cooling/energy.
We can regard the heat capacity and energy consumption as always on active cooling. 680 heat capacity is equivalent to 34 cooling at 0.5 heat dissipation. 34/2.4 = ~14.2.